source/binary.cpp

```diff
@@ -262,6 +263,9 @@ spv_result_t Parser::parseModule() {
                         << _.words[0] << "'.";
   }
   _.requires_endian_conversion = !spvIsHostEndian(_.endian);
+  _.native_words = std::make_unique<uint32_t[]>(_.num_words);
```

Comment:
Let's try to fix the logic without doubling copying the whole module, for everybody in all situations.